### PR TITLE
fixed linting for all the hbs files

### PR DESCRIPTION
**Pull request by fzngagan**

### CODE REVIEW

File: assets/javascripts/discourse/templates/admin-wizards-custom-fields.hbs [hunk 1]
```diff
@@ -14,23 +14,27 @@
...
-  component='custom_fields'}}
+  component="custom_fields"}}
 
 <div class="admin-wizard-container">
   {{#if customFields}}
     <table>
-      <tr>
-        {{#each fieldKeys as |key|}}
-          <th>{{i18n (concat "admin.wizard.custom_field." key ".label")}}</th>
+      <thead>
```
Comment (fzngagan) [inline assets/javascripts/discourse/templates/admin-wizards-custom-fields.hbs:22]: I've checked this table and its dropdowns are a too wide and look the same on the master branch. Here's a video of how they look on the master branch.

https://user-images.githubusercontent.com/19965650/114768668-d20a4900-9d86-11eb-8f56-86db6843adbb.mov

File: assets/javascripts/discourse/templates/components/wizard-custom-field.hbs [hunk 2]
```diff
@@ -1,227 +1,227 @@
...
+            placeholderKey="admin.wizard.translation_placeholder"}}
+        </div>
+      </div>
+      
+      {{#if validations}}
+        {{wizard-realtime-validations field=field validations=validations}}
+      {{/if}}
+    </div>
+  {{/if}}
+{{/if}}
```
Comment (fzngagan) [inline assets/javascripts/discourse/templates/components/wizard-custom-field.hbs:227]: This diff looks weird. The reason being, I used the `sed` command to replace single quotes with double quotes but I've carefully looked at the file and there's nothing to worry.

File: assets/javascripts/discourse/templates/admin-wizards-custom-fields.hbs (hunk 1 above)
Comment (angusmcleod) [reply assets/javascripts/discourse/templates/admin-wizards-custom-fields.hbs:22]: @fzngagan They're not like this on master, e.g. thepavilion.io is running the latest. Please check again.

Screen Shot 2021-04-15 at 10 16 18 AM

File: assets/javascripts/discourse/templates/components/wizard-links.hbs [hunk 3]
```diff
@@ -1,14 +1,14 @@
-<div class="wizard-header medium">{{{i18n header}}}</div>
+<div class="wizard-header medium">{{html-safe (i18n header)}}</div>
 
 <div class="link-list">
   {{#if anyLinks}}
     {{#each links as |l|}}
-      <div data-id='{{l.id}}'>
+      <div data-id={{l.id}}>
```
Comment (angusmcleod) [inline assets/javascripts/discourse/templates/components/wizard-links.hbs:6]: @fzngagan Why are we removing the quotes here?

File: assets/javascripts/wizard/templates/components/wizard-field-upload.hbs [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
+<label class="wizard-btn wizard-btn-upload-file {{if uploading "disabled"}}" tabindex={{field.tabindex}}>
   {{#if uploading}}
     {{wizard-i18n "wizard.uploading"}}
   {{else}}
     {{wizard-i18n "wizard.upload"}}
     {{d-icon "upload"}}
   {{/if}}
 
-  <input disabled={{uploading}} type="file" accept="{{field.file_types}}" style="visibility: hidden; position: absolute;" />
+  <input disabled={{uploading}} type="file" accept={{field.file_types}} style="visibility: hidden; position: absolute;" >
```
Comment (angusmcleod) [inline assets/javascripts/wizard/templates/components/wizard-field-upload.hbs:9]: @fzngagan And here

**Comment by fzngagan** [comment]
@angusmcleod 
I'm checking the styling breakage now. 
But regarding removing quotes, here's an ember twiddle to show how ember renders stuff inside `{{}}`
https://ember-twiddle.com/11825fea8591f9a55ef895f522af4bb7?openFiles=templates.application%5C.hbs%2C

That's the reason `ember-template-lint` considers the quotes redundant. :)

File: assets/javascripts/discourse/templates/admin-wizards-custom-fields.hbs (hunk 1 above)
Comment (fzngagan) [reply assets/javascripts/discourse/templates/admin-wizards-custom-fields.hbs:22]: @angusmcleod 
I tried to repro with the `hbs-lint` branch again but it looks fine now. Can you check on your side once. Weird.

**Pull request merged**